app/views/tasks/update.turbo_stream.erb

```diff
@@ -0,0 +1 @@
+<%= render "tasks/update", task: @task, project: @project %>
```

Comment:
@yshmarov because I want to use the same content for:

1. The controller response (`update.html.erb`), which is not a partial, but an action template.
2. The broadcasted partial (`_update.html.erb`).

It's just to reuse the same code in both.